docs/UI/round-corners.md

```diff
@@ -0,0 +1,4 @@
+# Round corners
+There is no native way to set a corner-radius for components / rectangle / anything at all.
+The best results can be achieved by using a MaskGroup. Keep in mind that MaskGroups can't use 9-patch-images.
```

Comment:
Could you add a code example of how to implement rounded corners using a `MaskGroup`?